dist/ng-wig.js

```diff
@@ -4,102 +4,102 @@
 angular.module('ngWig', ['ngwig-app-templates']);
 
 angular.module('ngWig')
-  .directive('ngWig', ["ngWigToolbar", function (ngWigToolbar) {
-
-    return {
-      scope: {
-        content: '=ngWig'
-      },
-      restrict: 'A',
-      replace: true,
-      templateUrl: 'ng-wig/views/ng-wig.html',
-      link: function (scope, element, attrs) {
-
-        scope.editMode = false;
-        scope.autoexpand = !('autoexpand' in attrs) || attrs['autoexpand'] !== 'off';
-        scope.toolbarButtons = ngWigToolbar.getToolbarButtons(attrs.buttons && string2array(attrs.buttons));
-
-        function string2array(keysString){
-          return keysString.split(',').map(Function.prototype.call, String.prototype.trim);
-        }
-
-        scope.toggleEditMode = function () {
-          scope.editMode = !scope.editMode;
-        };
-
-        scope.execCommand = function (command, options) {
-          if(scope.editMode ) return false;
+    .directive('ngWig', ["ngWigToolbar", function (ngWigToolbar) {
 
-          if (command === 'createlink') {
-            options = prompt('Please enter the URL', 'http://');
-            if(!options) {
-              return;
+        return {
+            scope: {
+                content: '=ngWig'
+            },
+            restrict: 'A',
+            replace: true,
+            templateUrl: 'ng-wig/views/ng-wig.html',
+            link: function (scope, element, attrs) {
+
+                scope.editMode = false;
```

Comment:
Yes, I messed up the **dist** directory by accident and tried to revert it back.